network-api/networkapi/mozfest/templates/mozfest/mozfest-base.html

```diff
@@ -27,18 +31,16 @@
 
 {% block content %}
   {% block secondary_nav %}{% endblock %}
-
   <div class="container cms">
-    <div class="row my-1 d-flex align-items-center">
-      <div class="pb-5 {% block bootstrap_width %}col-lg-8 offset-lg-2{% endblock %}">
-        {% for block in page.body %}
-          {% if block.block_type == 'heading' %}
-            <h1>{{ block.value }}</h1>
-          {% else %}
-            {% include_block block %}
-          {% endif %}
-        {% endfor %}
-      </div>
+    <div class="row my-1 d-flex align-items-start">
```

Comment:
@Pomax it's the `align-items-start` that's breaking the sticky CTA. We can actually remove both `d-flex` and `align-items-start`

Reply:
schweet